examples/opengl/OpenGL.cpp

```diff
@@ -3,6 +3,8 @@
 // Headers
 ////////////////////////////////////////////////////////////
 #include <SFML/Graphics.hpp>
+#include <cstdlib>
+#include <iostream>
```

Comment:
I would swap these includes so that C++ facilities are included before C facilities.